vm_image/build.sh

```diff
@@ -17,7 +17,7 @@ source '../scripts-common.sh'
 
 NEONVM_BUILDER_PATH='neonvm-builder'
 NEONVM_REPO='https://github.com/neondatabase/neonvm'
-NEONVM_VERSION='v0.4.5'
+NEONVM_VERSION='v0.4.6'
```

Comment:
is there a way that we can avoid the requirement of updating this value here and in the download deployments script so that upgrading a dependency can be done only in the go.mod file?

Reply:
duplicate, see below.